net.sf.eclipsecs.core/.classpath

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins">
+		<accessrules>
+			<accessrule kind="accessible" pattern="com/puppycrawl/tools/checkstyle/meta/**"/>
+		</accessrules>
+	</classpathentry>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry exported="true" kind="lib" path="lib/dom4j-2.1.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/metadata-gen-1.0-SNAPSHOT.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/reflections-0.9.10.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/snakeyaml-1.26.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/reflections-0.9.10.jar"/>
```

Comment:
This library is part of "-all.jar"
Please remove this line

Reply:
It didnt work during metadata-gen-all JAR for some reason, worked in sonar though.
So, I had to explicitly put it here.